pages/index.tsx

```diff
@@ -164,18 +164,18 @@ const HomePage = (props: any) => {
               <CodeWrapper>
                 <CodeExample
                   dangerouslySetInnerHTML={{
-                    __html: `yarn add <b>gatsby-plugin-tinacms</b>
+                    __html: `yarn add <b>tinacms</b>
 
-module.exports = {
-  <span>// ...</span>
-  plugins: [
-    '<b>gatsby-plugin-tinacms</b>',
-    <span>// ...</span>
-  ],
-};
+import { <b>withTina</b> } from 'tinacms'
+
+function MyApp({ Component, pageProps }) {
+  return <Component {...pageProps} />
+}
 
-export <b>WithTina</b>( <b>Component</b> );
-                  `,
+export default <b>withTina</b>(MyApp, {
+  <b>enabled: true</b>,
+  <b>sidebar: true</b>,
+})`,
```

Comment:
I think `Component` is messing up the styling. And some other parts of this aren't rendering in the example.

Reply:
@kendallstrautman What's wrong with the styling on your end?

Here's what I see:
<img width="561" alt="Capture d’écran 2020-11-16 à 17 28 40" src="https://user-images.githubusercontent.com/103008/99280075-3e613a80-2831-11eb-85ea-f3477bcfc34c.png">